lib/internal/test_runner/runner.js

```diff
@@ -114,16 +116,24 @@ function getRunArgs({ path, inspectPort }) {
 }
 
 
-function runTestFile(path, root, inspectPort) {
+function runTestFile(path, root, inspectPort, filesWatcher) {
   const subtest = root.createSubtest(Test, path, async (t) => {
     const args = getRunArgs({ path, inspectPort });
+    const stdio = ['pipe', 'pipe', 'pipe'];
+    let env = process.env;
+    if (filesWatcher) {
+      stdio.push('ipc');
+      env = { ...process.env, WATCH_REPORT_DEPENDENCIES: '1' };
```

Comment:
```suggestion
    const env = { ...process.env };
    if (filesWatcher) {
      stdio.push('ipc');
      env.WATCH_REPORT_DEPENDENCIES = '1';
```
I recommend doing this instead to dereference/copy `process.env` by default instead of having to dereference/copy it when this condition is met, it would also help reduce code when additional properties/methods need to be assigned later on.